### PR TITLE
Quick fix not being applied

### LINKED ISSUE #171: Quick fix not being applied
VS Code version: 1.84.1
Tools for MicroProfile version: v0.10.0 and v0.11.2023102104
OS: Windows 11

When attempting to apply a quick fix to add `@Liveness` or `@Readiness` annotations, no action occurs. Refer to video below.

https://github.com/redhat-developer/vscode-microprofile/assets/5885743/02a099b5-3c9c-4510-82a5-205be7967830

### PR DESCRIPTION
**Pull request by angelozerr**
Quick fix not being applied

Fixes #171

### CODE REVIEW

**Comment by angelozerr** [comment]
This PR fixes https://github.com/redhat-developer/vscode-microprofile/issues/171 by using LSP CodeAction as `microprofile.applyCodeAction` as arguments, because LSP CodeAction is a simple JSON object although before we use VSCode CodeAction which are complex object and cannot be serialized correctly when the code action is set as arguments of the command.

I had to change the language server client version, because I need asCodeActionSync function.

I have tested with several code actions kind:

 * code action with text edit resolve (code action to update Java file like the original issue)
 * code action with text edit already resolved (ex : you write `quarkus.jni.enable=fal` in an application.properties which reports an error to `fal` and code action fix it with true, false.)
 * code action which execute a simple command (ex : you write `foo.bar=baz` in application.properties, which reports an error to `foo.baz` and you can exclude the validation, here it uses a simple command which update the settings).

**Comment by angelozerr** [comment]
I don't know why windows build is falling https://github.com/redhat-developer/vscode-microprofile/actions/runs/6810212978/job/18518488267 ?

@datho7561 have you some idea?

File: package-lock.json [hunk 1]
```diff
@@ -365,7 +365,7 @@
     },
     "@types/chai-fs": {
       "version": "2.0.2",
-      "resolved": "https://registry.npmjs.org/@types/chai-fs/-/chai-fs-2.0.2.tgz",
+      "resolved": "https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/@types/chai-fs/-/chai-fs-2.0.2.tgz",
```
Comment (rgrunber) [inline package-lock.json:368]: Make sure to update these to use `registry.npmjs.org`.

**Comment by datho7561** [comment]
"Failed to get VS Code archive location" is output on Roland's latest run. My guess is that the download is timing out

**Comment by rgrunber** [comment]
See https://github.com/microsoft/vscode/issues/197554 . Looks like `windows-latest` is pulling a 32-bit build. Is a 64bit builder available ?

**Pull request merged**